apps/cookbook/src/app/showcase/modal-experimental-showcase/modal-experimental-showcase.component.html

```diff
@@ -0,0 +1,185 @@
+<p><b>This is an experimental feature and is subject to change.</b></p>
```

Comment:
Not mega important, but consider: 

```suggestion
<kirby-flag themeColor="warning">Experimental component - subject to change</kirby-flag>

```